android-sdk/tasks/main.yml

```diff
@@ -5,20 +5,15 @@
   set_fact: buildfarm_templates_dir="/opt/rhmap/{{ rhmap_version }}/buildfarm/templates"
   when: buildfarm_templates_dir is not defined
 
--
-  name: "Set default config directory"
-  set_fact: config_file_dir="/opt/rhmap/{{ rhmap_version }}/rhmap-installer/roles/feedhenry.buildfarm-installer/templates"
-  when: config_file_dir is not defined
-
 - name: "Create project"
-  command: "oc new-project {{ project_name }}"
+  command: "oc new-project {{ project_name }} --namespace={{ project_name }}"
   register: create_project_result
-  failed_when: create_project_result.stderr and create_project_result.stderr != '' and 'already_exists' not in create_project_result.stderr
+  failed_when: create_project_result.stderr and create_project_result.stderr != '' and 'already exists' not in create_project_result.stderr
```

Comment:
What about using `AlreadyExists` here as that's the title of the failure. That probably couldn't change without being backwards incompatible whereas the `already exists` in the comment could. WDYT?